BinaryObjectScanner/Protection/HexalockAutoLock.cs

```diff
@@ -57,7 +57,23 @@ public class HexalockAutoLock : IExecutableCheck<PortableExecutable>, IPathCheck
                 if (strs.Exists(s => s.Contains("mfint.dll")))
                     return "Hexalock Autolock";
             }
-
+            // Get the code/CODE section strings, if they exist
```

Comment:
Please add newlines after the previous `if` block. Same below on the UPX check.

Reply:
Fixed.